tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -240,11 +348,40 @@ def test_wazuh_db_messages_agent(restart_wazuh, clean_registered_agents, configu
                               for case in module_data]
                          )
 def test_wazuh_db_messages_global(connect_to_sockets_module, restart_wazuh, test_case):
-    """Check that every input global message in wazuh-db socket generates the adequate output to wazuh-db socket.
-
-    Args:
-        test_case(list): List of test_case stages (dicts with input, output and stage keys).
-    """
+    '''
+    description: Check that every input global message in wazuh-db socket generates the adequate output to wazuh-db
+                 socket. To do this, query the socket with a command taken from the input list of stages (test_case,
```

Comment:
I would write the first sentence like this.
```suggestion
    description: Check that every global input message in wazuh-db socket generates the proper output to wazuh-db
                 socket. To do this, it performs a query to the socket with a command taken from the input list of stages (test_case,
```

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)